.github/workflows/github-actions.yml

```diff
@@ -1,26 +1,12 @@
-name: assignment-test
-on: 
-    push:
-        tags-ignore:
-            - '*'
-        branches:
-            - '*'
+name: assignment-autotest-test
+on: [push]
```

Comment:
@ritikar97 don't overwrite this file with old content from previous semesters, this will cause problems with future assignments.